tests/testthat/test-ratio_calculations.R

```diff
@@ -41,17 +49,169 @@ describe("multiple_matrix_ratios function", {
     data <- data.frame(
       USUBJID = rep("A", 12),
       TIME = c(0, 1, 2, 3, 0, 1, 2, 4, 0, 1, 3, 4),
-      MATRIX = c("BLOOD", "PLASMA", "BRAIN", "LIVER", "BLOOD", "PLASMA", "BRAIN", "LIVER",
-                 "BLOOD", "PLASMA", "BRAIN", "LIVER"),
+      MATRIX = c(
+        "BLOOD", "PLASMA", "BRAIN", "LIVER", "BLOOD", "PLASMA", "BRAIN", "LIVER",
+        "BLOOD", "PLASMA", "BRAIN", "LIVER"
+      ),
       CONC = c(10, 25, 5, 12, 20, 30, 10, 18, 15, 40, 8, 16),
       UNITS = rep("ng/mL", 12)
     )
 
-    result <- multiple_matrix_ratios(data, "MATRIX", "CONC", "UNITS",
-                                     c("TIME", "USUBJID"), c("BLOOD", "BRAIN"),
-                                     c("PLASMA", "LIVER"))
+    result <- multiple_matrix_ratios(
+      data, "MATRIX", "CONC", "UNITS",
+      c("TIME", "USUBJID"), c("BLOOD", "BRAIN"),
+      c("PLASMA", "LIVER")
+    )
 
     expect_true(nrow(result) < 8)
   })
+})
+
+describe("calculate_ratios function", {
```

Comment:
nitpick: Just the name of the function is enough

```suggestion
describe("calculate_ratios", {
```